Content.Server/GameTicking/Rules/Components/LoadMapRuleComponent.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Server.Maps;
+using Content.Shared.Whitelist;
+using Robust.Shared.Map;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Utility;
+
+namespace Content.Server.GameTicking.Rules.Components;
+
+/// <summary>
+/// This is used for a game rule that loads a map when activated.
+/// </summary>
+[RegisterComponent]
+public sealed partial class LoadMapRuleComponent : Component
+{
+    [DataField]
+    public MapId? Map;
+
+    [DataField]
+    public ProtoId<GameMapPrototype>? GameMap ;
+
+    [DataField]
+    public string? MapPath;
+
+    [DataField]
+    public List<EntityUid> MapGrids = new();
+
```

Comment:
I doubt you added code for it but probably good to also susbcribe to gridsplitting and handle it too maybe?